index.html

```diff
@@ -51,6 +51,200 @@ <h2>
+                    <div>
+                        <ul>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                    <p style="padding-bottom: 78px;">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="languages">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div style="padding-top: 24px; padding-bottom: 20px;">
+                        <button>See project</button>
+                    </div>
+
+               </div>
+                
+            </div>
+
+        </section>
+        <section class="work-section last-work-section">
+            <div class="page-container">
+                <div class="img.wrap">
+                    <img src="images/Snapshoot Portfolio (3).svg">
+                </div>
+               <div class="details">
+                    <div>
+                        <h2>Multi-Post Stories</h2>
+                    </div>
+                    <div>
+                        <ul>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                    <p style="padding-bottom: 78px;">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="languages">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div style="padding-top: 24px; padding-bottom: 20px;">
+                        <button>See project</button>
+                    </div>
+
+               </div>
+                
+            </div>
+
+        </section>
+        <section class="about-myself">
+            <div>
+                <h1>
+                    About Myself
+                </h1>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="socials">
+                <h4>
+                    LET’S CONNECT
+                </h4>
+                <ul>
+                    <li>
+                        <a style="width:19.99px; height:19.96px" href="#"><img src="images/Twitter.svg"></a>
```

Comment:
- Kindly, use the external style CSS as your precious code reviewer suggested 👌👌 Always use the class attribute for multiple elements (do not use ID selector for multiple elements). 👍 👍